src/status_im2/contexts/onboarding/identifiers/profile_card/view.cljs

```diff
@@ -0,0 +1,73 @@
+(ns status-im2.contexts.onboarding.identifiers.profile-card.view
+  (:require [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [react-native.reanimated :as reanimated]
+            [react-native.fast-image :as fast-image]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]
+            [quo2.components.profile.profile-card.style :as profile-card-style]
+            [quo2.components.avatars.user-avatar.view :as user-avatar]
```

Comment:
please import as `[quo2.core :as quo]` and then use as `quo/text`, `quo/user-avatar`

Reply:
Resolved